lib/reporter.ts

```diff
@@ -201,7 +201,9 @@ export class CucumberJsJsonReporter extends WDIOReporter {
      */
     public onRunnerEnd (): void {
         const jsonFolder = resolve( process.cwd(), this.options.jsonFolder );
-        const jsonFile = resolve( jsonFolder, `${this.report.feature.id}.json` );
+        const date = new Date(Date.now()).toISOString().replace(/:/g,"-").split(".")[0];
+        const browser = this.instanceMetadata.browser.name;
```

Comment:
Let's not add the browsername in her. There are multiple reasons:
- this module can be used for dekstop, mobile and electron apps. Not all have the browser capability so this would break
- imho the file name itself should be unique and the timestamp is a good way, adding the browserName makes it and more complex (see point above) and normally the JSON files are used for parsing in reporters. There the filename doesn't matter
- if we want to make it more unique I'd love to go for the solution from @bhaslamb because `String( Date.now() + Math.random() ).replace( '.','' )` will result in a unique string, the `new Date(Date.now()).toISOString().replace(/:/g,"-").split(".")[0]` us still not unique enough because it only shows the minutes, It might be even more simple by using `+new Date` which will result in something like `1643788925816`

Can we also please add Unit tests for this so we have this covered?

Reply:
Ok, removing the browser name and will try either `+new Date`  or `String( Date.now() + Math.random() ).replace( '.','' )`.

Will need some guidance on Unit tests.

Reply:
> Ok, removing the browser name and trying `const date = new Date().getTime();` which results in suffix like _1643821183165
> 
> Will need some guidance on Unit tests.
--> Could you please take a look at this closed PR? It generates unique file name with fixed failed unit test as well.
https://github.com/webdriverio-community/wdio-cucumberjs-json-reporter/pull/84

Reply:
@osmolyar - if you are agree with my change, i will create PR for this together with unit test otherwise i don't want to make conflicts on this.

Reply:
@bhaslamb that sounds great!  Please feel free to create a PR and unit test.  Thanks a lot.

Reply:
@osmolyar  @wswebcreation  - Reopened this PR to generate unique file for every run. 
https://github.com/webdriverio-community/wdio-cucumberjs-json-reporter/pull/84

Reply:
thanks for all the work @osmolyar and sorry for not being able to support you properly the last days. I've been very busy
@bhaslamb , thanks for your support!!!

I'll close this PR and will look at #84